ipn/ipnlocal/serve.go

```diff
@@ -56,41 +56,52 @@ type serveHTTPContext struct {
 	DestPort uint16
 }
 
-// serveListener is the state of host-level net.Listen for a specific (Tailscale IP, serve port)
+// localListener is the state of host-level net.Listen for a specific (Tailscale IP, serve port)
```

Comment:
s/serve port/port/ ?

but then mention the two things that this is used for, for both "tailscale serve" and the web client?

Reply:
done